### PR TITLE
Translation + Transliteration for Dhan Slides and Mool mantar

### PR DESCRIPTION
**Pull request by AkalUstat**
~~in progress~~ complete

### CODE REVIEW

File: www/js/strings.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    "gurmukhi": "<> siq nwmu krqw purKu inrBau inrvYru Akwl mUriq AjUnI sYBM gur pRswid ]",
+    "english": "One Universal Creator God. The Name Is Truth. Creative Being Personified. No Fear. No Hatred. Image Of The Undying, Beyond Birth, Self-Existent. By Guru's Grace ~",
+    "punjabi": "ਅਕਾਲ ਪੁਰਖ ਇੱਕ ਹੈ, ਜਿਸ ਦਾ ਨਾਮ 'ਹੋਂਦ ਵਾਲਾ' ਹੈ ਜੋ ਸ੍ਰਿਸ਼ਟੀ ਦਾ ਰਚਨਹਾਰ ਹੈ, ਜੋ ਸਭ ਵਿਚ ਵਿਆਪਕ ਹੈ, ਭੈ ਤੋਂ ਰਹਿਤ ਹੈ, ਵੈਰ-ਰਹਿਤ ਹੈ, ਜਿਸ ਦਾ ਸਰੂਪ ਕਾਲ ਤੋਂ ਪਰੇ ਹੈ, (ਭਾਵ, ਜਿਸ ਦਾ ਸਰੀਰ ਨਾਸ-ਰਹਿਤ ਹੈ), ਜੋ ਜੂਨਾਂ ਵਿਚ ਨਹੀਂ ਆਉਂਦਾ, ਜਿਸ ਦਾ ਪ੍ਰਕਾਸ਼ ਆਪਣੇ ਆਪ ਤੋਂ ਹੋਇਆ ਹੈ ਅਤੇ ਜੋ ਸਤਿਗੁਰੂ ਦੀ ਕਿਰਪਾ ਨਾਲ ਮਿਲਦਾ ਹੈ।",
+    "translit": " ikOankaar sat naam karataa purakh nirabhau niravair akaal moorat ajoonee saibha(n) gur prasaadh ||"
+  },
+  "waheguru": "vwihgurU",
+  "dhanSlides": [
+    {
+      "gurmukhi": "DMn gurU nwnk dyv jI",
+      "english": "Dhan Guru Nanak Dev Ji",
```
Comment (maneetpaul) [inline www/js/strings.json]: @saintsoldierx do we want to translate "Dhan" for the English or just leave it like this? 

"Blessed is Guru Nank Dev Ji" ?

File: www/js/strings.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    },
+    {
+      "gurmukhi": "DMn gurU goibMd isMG swihb jI",
+      "english": "Dhan Guru Goind Singh Sahib Ji",
+      "punjabi": "",
+      "translit": ""
+    },
+    {
+      "gurmukhi": "DMn gurU gRMQ swihb jI",
+      "english": "Dhan Guru Granth Sahib Ji",
```
Comment (maneetpaul) [inline www/js/strings.json]: We should "Sri" since it is almost always said Dhan Sri Guru Granth Sahib Jee

File: www/js/strings.json (hunk 1 above)
Comment (AkalUstat) [reply www/js/strings.json]: i think if we're going to do translations, 'dhan' should probs translate to 'great'

File: www/js/strings.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    },
+    {
+      "gurmukhi": "DMn gurU AMgd dyv jI",
+      "english": "Great is Guru Angad Dev Ji",
+      "punjabi": "",
+      "translit": ""
+    },
+    {
+      "gurmukhi": "DMn gurU Amrdwsu swihb jI",
+      "english": "Great is Guru Amar Das Sahib Ji",
```
Comment (maneetpaul) [inline www/js/strings.json]: Change Das to Daas

File: www/js/strings.json [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+      "punjabi": "",
+      "translit": ""
+    },
+    {
+      "gurmukhi": "DMn gurU rwmdws swihb jI",
+      "english": "Great is Guru Ram Das Sahib Ji",
```
Comment (maneetpaul) [inline www/js/strings.json]: Change Das to Daas

**Comment by AkalUstat** [comment]
@inderpreetsingh ji or @tsingh777 ji could one of you guys check to make sure the code is correct?

File: www/js/insert-slide.js [hunk 5]
```diff
@@ -3,13 +3,28 @@ const electron = require('electron');
 
 const analytics = electron.remote.getGlobal('analytics');
 const tingle = require('./vendor/tingle');
-const strings = require('./strings');
+const strings = require('./strings.json');
```
Comment (tsingh777) [inline www/js/insert-slide.js]: `setLangSliderVal` function needs to be updated as well. `strings.announcemenetPlaceholder` is undefined.

File: www/js/controller.js [hunk 6]
```diff
@@ -575,6 +575,12 @@ module.exports = {
     global.platform.ipc.send('show-empty-slide');
     global.platform.ipc.send('show-text', { text, isGurmukhi, isAnnouncement });
   },
+  sendMimicShabad(mimicShabad) {
```
Comment (tsingh777) [inline www/js/controller.js]: please call the function `sendTextWithTranslations` instead of "sendMimicShabad"

File: app.js [hunk 7]
```diff
@@ -451,7 +451,21 @@ ipcMain.on('show-line', (event, arg) => {
     createBroadcastFiles(arg);
   }
 });
-
+ipcMain.on('mimic-shabad', (event, arg) => {
```
Comment (tsingh777) [inline app.js]: please call the event `show-text-with-translations` instead of "mimic-shabad"

File: www/js/controller.js (hunk 6 above)
Comment (tsingh777) [inline www/js/controller.js]: `menu.js` also should call this in the `dhanGuruSlideButton` click handler.

Comment (AkalUstat) [reply www/js/controller.js]: What does that mean?

Comment (AkalUstat) [reply www/js/controller.js]: Like it should have an event?

File: www/js/strings.json (hunk 1 above)
Comment (tarunsingh5) [reply www/js/strings.json]: +1 to "Great"

Comment (tarunsingh5) [reply www/js/strings.json]: should we add "Sahib"

File: www/js/strings.json (hunk 2 above)
Comment (tarunsingh5) [reply www/js/strings.json]: +1

**Comment by AkalUstat** [comment]
@tsingh777 still need help with menu.jd

File: www/js/strings.json (hunk 1 above)
Comment (saintsoldierx) [reply www/js/strings.json]: "Great" is fine. Please do add Sahib. Normally when sung, we sing "Dhan Guru Nanak Dev Sahib Jee, Dhan Guru Angad Dev Sahib Ji" etc..

File: www/js/strings.json (hunk 2 above)
Comment (saintsoldierx) [reply www/js/strings.json]: Yes, only when Sri Guru Granth Sahib Ji is sung. Maybe not for others.

**Comment by AkalUstat** [comment]
@saintsoldierx @tarunsingh5 could you take a final look

**Pull request merged**